wagtailio/static/sass/components/_buttons.scss

```diff
@@ -28,12 +28,27 @@
     }
 
     &--dark {
-        background-color: $color--indigo;
+        border-color: $color--white;
+        background-color: transparent;
         color: $color--white;
 
-        &:hover {
+        &:hover,
+        &:focus {
             background-color: $color--white;
-            color: $color--off-black;
+            border-color: $color--white;
+            color: $color--indigo;
+        }
+    }
+
+    &--secondary {
+      background-color: $color--indigo;
```

Comment:
Same here with the indentation. Might just be the way it's displaying here though